src/types/index.tsx

```diff
@@ -20,7 +20,9 @@ export interface Project {
   lookingFor: string;
   description: string;
   contributors: Array<string>;
-  tags?: Array<string>;
+  tags: Array<string>;
+  url: string;
```

Comment:
this should be `github_url` to match the data coming from API.

Reply:
👍 
Though `github_url` or `githubURL` / `githubUrl`? 
Because `lookingFor` 🤔

Reply:
Updated to `githibUrl`